doc/qos/SONiC_ROCEv2_support.md

```diff
@@ -0,0 +1,488 @@
+# Feature Name
+RDMA (Remote Direct Memory Access) over Converged Ethernet (ROCEv2) support

```

Comment:
Please rephrase it to "Configuration Management support for ROCEv2" or "Management Framework support for ROCEv2"

Reply:
The scope is end-to-end ROCEv2 testing, will rephrase accordingly.